code/game/machinery/computer/buildandrepair.dm

```diff
@@ -219,6 +219,9 @@
 /obj/item/circuitboard/prisoner
 	name = "Circuit board (Prisoner Management)"
 	build_path = /obj/machinery/computer/prisoner
+/obj/item/circuitboard/brigcells

```

Comment:
Spacing between this and previous line please

Reply:
@variableundefined It’s like that for the whole file. Unless you want me to space the **whole** file it’s staying as it is